rspirv/Cargo.toml

```diff
@@ -22,10 +22,7 @@ clippy = { version = "0.0", optional = true }
 derive_more = "0.99"
 fxhash = "0.2"
 num-traits = "0.2"
-
-[dependencies.spirv]
-version = "=0.1"
-path = "../spirv"
+spirv = { version = "=0.1.0+1.5.4", path = "../spirv" }
```

Comment:
A dependency shouldn't use the `+metadata` bit though, I hope `cargo-release` understands and strips that off?

Reply:
It ouput's like this which seems to make sense: `Fixing rspirv's dependency on spirv to '= 0.1.1' (from '= 0.1.0')"`

Reply:
@Jasper-Bekkers Does that mean it's already bumping the version to 0.1.1 before releasing? We should teach `cargo-release` that `0.1.0` is our first release that isn't actually out yet. Maybe we should set it to `0.0.0` and pass `0.1.0` or `1.0.0` as argument to `cargo-release` instead?

Reply:
Good point! :-)